controllers/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -225,15 +219,27 @@ func (r *ApplicationSnapshotEnvironmentBindingReconciler) Reconcile(ctx context.
 				Value: env.Value,
 			})
 		}
+		componentResources := corev1.ResourceRequirements{}
+		if component.Configuration.Resources != nil {
+			componentResources = *component.Configuration.Resources
+		}
 
-		gitopsgenEnv := gitopsgenv1alpha1.Environment{
-			Spec: gitopsgenv1alpha1.EnvironmentSpec{
-				Configuration: gitopsgenv1alpha1.EnvironmentConfiguration{
-					Env: environmentConfigEnvVars,
-				},
-			},
+		kubeLabels := map[string]string{
+			"app.kubernetes.io/name":       componentName,
+			"app.kubernetes.io/instance":   component.Name,
+			"app.kubernetes.io/part-of":    applicationName,
+			"app.kubernetes.io/managed-by": "kustomize",
+			"app.kubernetes.io/created-by": "application-service",
+		}
```

Comment:
can you help remind me, where were these labels coming from previously?

Reply:
could you point to the change, which removed these?

Reply:
It's the last change in the file generate.go

https://github.com/redhat-developer/gitops-generator/pull/20/files#diff-afe63b2c70508656d0bfa0f0acb5d9bfc188b3e65ab7bf007f6e0b2cf52ad46f

We discussed that the labels should be customizable.

If you don't set it, the only difference is that the default is to set this:

		"app.kubernetes.io/name":       options.Name,
		"app.kubernetes.io/instance":   options.Name,

However in the original HAS code, component.Name is used for the `app.kubernetes.io/instance` and the component's name was used for the `app.kubernetes.io/name`.  Hence, in HAS code, you have to customize these labels in order to maintain the previous behaviour.